src/Snppts/Snippets/CarouselIntroPage.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using Snppts.Authors;
+using Snppts.Extensions;
+using Snppts.Infrastructure;
+
+namespace Snppts.Snippets
+{
+    public class CarouselIntroPage : IAmASnippet
+    {
+        public string Slug => "carousel-intro-page";
+        public string Title => "Carousel Intro Page";
+
+        public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("LucioMSP/Xamarin.Forms.Examples/tree/master/CarouselIntroPage");
+
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => false;
+
+        public string Description => "Simple example of an introductory page in Xamarin.Forms.";
+
+        public IAmAnAuthor AuthorInfo => new VicenteGuzman();
+
+        public IEnumerable<Uri> ImageUris => new List<Uri>
+        {
+            new Uri("https://github.com/LucioMSP/Xamarin.Forms.Examples/blob/master/CarouselIntroPage/ScreenShots/ScreenShot01.png"),
```

Comment:
Can you link to the direct screenshots for all of these please?

Reply:
Done

Reply:
I think you misunderstood. It can definitely be multiple screenshots, just link to the so-called raw version. If you look at the link you had here earlier it would show the screenshot on the GitHub page. We would need the url to the direct screenshot for showing it on our page. 

For reference have a look at other snippets

Reply:
He thinks that if he had misunderstood, he already corrected it.